File: 1-1.php
<?php

$filename = 'pandas/edu_bigdata_imp1.csv';
$delimiter = ',';
$enclosure = '"';
$escape = '\\';

$file = fopen($filename, 'r');

$data = [];
$header = fgetcsv($file, 0, $delimiter, $enclosure, $escape);

$pseudoIDIndex = array_search('PseudoID', $header);
$reviewSNIndex = array_search('dp001_review_sn', $header);

$uniqueValues = [];

while (($row = fgetcsv($file, 0, $delimiter, $enclosure, $escape)) !== false) {
    if ($row[$pseudoIDIndex] == 39) {
        $uniqueValues[] = $row[$reviewSNIndex];
    }
}

fclose($file);

$uniqueCount = count(array_unique($uniqueValues));

echo $uniqueCount;

?>

File: 1-1.py
import pandas as pd

df = pd.read_csv('pandas\edu_bigdata_imp1.csv', encoding = 'big5', low_memory = False)
df_filtered = df[df['PseudoID'] == 39]
unique_value = df_filtered['dp001_review_sn'].unique()
print(len(unique_value))